Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
Your .Rmd files should go into folders under the 'inst' folder. See how they are organized here: https://github.com/USGS-R/repgen for a good example of how we should organize the package.